### DISCUSSION TITLE
Installer not working with Galera/Mysql Cluster due to auto increment not being one

### DISCUSSION THREAD

**Discussion by xeroc** [requarks/wiki#5923] [Error / Bug Report]
During installation of wikijs, I ran into this assert:

https://github.com/requarks/wiki/blob/97a744556b8afa8687f9fcf1efea410a74d70217/server/setup.js#L254

After some research, I noticed that the id of my admin group is correct (1), but the id of the guest group is incorrect (3). The reason is that I use a galera/mariadb cluster:

* https://stackoverflow.com/questions/63921999/auto-increment-jumps-by-3-and-not-1
* https://mariadb.org/auto-increments-in-galera/

Not sure why the guest group id has to be 2 and cannot be three but maybe create those entries manually and set the auto increment value manually afterwards.

**Comment by xeroc** [comment]
anyone can recommend how to deal with this?

**Comment by ret1** [comment]
i've got the same error on mariadb
